src/test/java/ru/training/at/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,204 @@
+
+        //Locating the login field and filling it in
+        WebElement loginField = webDriver.findElement(By.id("name"));
+        loginField.sendKeys(userName);
+
+        //Locating the password field and filling it in
+        WebElement passwordField = webDriver.findElement(By.id("password"));
+        passwordField.sendKeys(password);
+
+        //Locating the enter button and performing a clock on it
+        WebElement enterButton = webDriver.findElement(By.className("fa-sign-in"));
+        enterButton.click();
+    }
+
+    public void checkUsernameAfterLogin() {
+        //Locating the username element with id
+        WebElement displayedName = webDriver.findElement(By.id("user-name"));
+
+        //SoftAsserting that the username is displayed
+        softAssertObject.assertTrue(displayedName.isDisplayed());
+
+        //SoftAsserting that the displayed username matches the expected username
+        String expectedUsername = "ROMAN IOVLEV";
+        softAssertObject.assertEquals(displayedName.getText(), expectedUsername);
+    }
+
+    public void checkHeaderItems() {
+        //Locating the elements with CSS selectors
+        WebElement homeButton = webDriver.findElement(By.cssSelector("ul.uui-navigation>li:nth-child(1)>a[href='index.html']"));
+        WebElement contactFormButton = webDriver.findElement(By.cssSelector("ul.uui-navigation>li:nth-child(2)>a[href='contacts.html']"));
+        WebElement serviceButton = webDriver.findElement(By.cssSelector("ul.uui-navigation>li:nth-child(3).dropdown>a"));
+        WebElement metalsAndColorsButton = webDriver.findElement(By.cssSelector("ul.uui-navigation>li:nth-child(4)>a[href='metals-colors.html']"));
+
+        // SoftAsserting the visibility of the elements
+        softAssertObject.assertTrue(homeButton.isDisplayed());
+        softAssertObject.assertTrue(contactFormButton.isDisplayed());
+        softAssertObject.assertTrue(serviceButton.isDisplayed());
+        softAssertObject.assertTrue(metalsAndColorsButton.isDisplayed());
+
+        // SoftAsserting button texts
+        softAssertObject.assertEquals(homeButton.getText(), "HOME");
+        softAssertObject.assertEquals(contactFormButton.getText(), "CONTACT FORM");
+        softAssertObject.assertEquals(serviceButton.getText(), "SERVICE");
+        softAssertObject.assertEquals(metalsAndColorsButton.getText(), "METALS & COLORS");
+    }
+
+    public void checkIndexPageIcons() {
+        //Locating elements with CSS selectors
+        WebElement microscopeIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-practise"));
+        WebElement headsetIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-custom"));
+        WebElement screenIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-multi"));
+        WebElement rocketIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-base"));
+
```

Comment:
Could be used `List<WebElement>` and loop for the assertion

Reply:
Great idea! I will do that.